docs/reference/components/sources/internal_metrics.cue

```diff
@@ -0,0 +1,44 @@
+package metadata
+
+components: sources: internal_metrics: {
+	title:       "Internal Metrics"
+	description: "The internal metrics source exposes metrics emitted by the running Vector instance (as opposed to components in its topology)."
+
+	classes: {
+		commonly_used: true
+		delivery:      "at_least_once"
+		deployment_roles: ["aggregator", "daemon", "sidecar"]
+		development:   "beta"
+		egress_method: "batch"
+	}
+
+	features: {
+		collect: {
+			checkpoint: enabled: false
+			from: {
+				name:     "Vector instance"
+				thing:    "a \(name)"
+				url:      urls.vector_docs
+				versions: ">= 0.11.0"
+			}
+		}
+		multiline: enabled: false
+	}
+
+	support: {
+		platforms: {
+			"aarch64-unknown-linux-gnu":  true
+			"aarch64-unknown-linux-musl": true
+			"x86_64-apple-darwin":        true
+			"x86_64-pc-windows-msv":      true
+			"x86_64-unknown-linux-gnu":   true
+			"x86_64-unknown-linux-musl":  true
+		}
+
+		notices: []
+		requirements: []
+		warnings: []
+	}
+
+	output: metrics: _internal_metrics
```

Comment:
Given my comment on https://github.com/timberio/vector/pull/4820/files#r522330914, this could be rewritten as:

```
output: metrics: internal_metrics.telemetry.metrics._events
```

Where `_events` is defined in `components.cue` as:

```
// ...
telemetry: metrics: {
	_events: {
		invalid_record_bytes_total: {...}
		invalid_record_total: {...}
	}
}
// ...
```

Does that make sense?